### ISSUE TITLE
Matching between runs with a library set does not seem to add much? (not an issue)

### ISSUE THREAD

**Issue by jadv12** [Nesvilab/FragPipe#525]
Hi there,

I just want to say first that this is **not an issue or bug.** Just hoping to get some thoughts/opinions. 

I have been following the MaxQuant group's method of incorporating a subset of depleted and fractionated samples as a means to counteract the high missingness in plasma proteomics using the MBR setting. My proposed experiment would include a total of 226 samples:
- 200 experimental samples
- 16 depleted fractions (library runs)
- 10 technical replicates

I was hoping replicate this boost in complete data with Fragpipe (which has been amazing at identifying more total proteins AND in a much faster manner).

Now, my runs have been getting far less proteins and peptides per individual sample than MQ. I've been searching 3 replicates with and without the library runs (8 fractions). All with min. ions of 1 (due to low number of samples) and FDR of 1%. I was also curious to see what parameter optimization does as well as the "lib" functionality when you put lib in the experiment name (saw this in https://github.com/Nesvilab/FragPipe/issues/343#issuecomment-813696409).

Summary of quantified proteins and peptides (by MaxLFQ) for the different parameters
![image](https://user-images.githubusercontent.com/93340685/141538542-a6dbabe6-21fa-4695-a7aa-22f4806abe55.png)

I found it interesting that adding the library runs reduced the number of quantified proteins and peptides. Even more so when using the "lib" experiment names. Maybe I'm misinterpreting the data?

**Comment by jadv12** [comment]
**PNGs when searching the replicates only**

![20210901_timsTOF_Evo_22min_CDS_VanJ_S1_S3-B9_1_509_model](https://user-images.githubusercontent.com/93340685/141539492-4597775b-5780-4089-8412-401dea1bcb00.png)
![20210901_timsTOF_Evo_22min_CDS_VanJ_S2_S3-D6_1_532_model](https://user-images.githubusercontent.com/93340685/141539606-efc241cf-24f7-413c-a2d7-5b62df7ae0bb.png)
![20210901_timsTOF_Evo_22min_CDS_VanJ_S3_S3-F3_1_555_model](https://user-images.githubusercontent.com/93340685/141539631-4e491ac0-9751-429c-9889-24a8395292d0.png)

**PNG when searching the replicates with the library runs**
![20210901_timsTOF_Evo_22min_CDS_VanJ_S1_S3-B9_1_509_model](https://user-images.githubusercontent.com/93340685/141539689-e19bd351-f266-45e1-b0d3-8a788797f847.png)
![20210901_timsTOF_Evo_22min_CDS_VanJ_S2_S3-D6_1_532_model](https://user-images.githubusercontent.com/93340685/141539694-3b4de9c9-221a-4921-b1cb-a2baa5eea737.png)
![20210901_timsTOF_Evo_22min_CDS_VanJ_S3_S3-F3_1_555_model](https://user-images.githubusercontent.com/93340685/141539699-0152675f-e9d1-4c14-bdd1-931c011b8cc2.png)

Maybe I need to change the parameters such that more target ions score > 0? If that is possible?

**Comment by fcyu** [comment]
In https://github.com/Nesvilab/FragPipe/issues/504 , we just convinced you that FragPipe quantified more proteins with 100% MBR-FDR (which is equivalent to what MaxQuant has). You also agreed that with 100% MBR-FDR, there are many false positives, and should use 1-5% MBR-FDR.

As to your question "I found it interesting that adding the library runs reduced the number of quantified proteins and peptides. Even more so when using the "lib" experiment names. Maybe I'm misinterpreting the data?"

There is no guarantee that more runs will result in more quantified proteins and peptides if you have FDR control. Same logic as bigger database won't always result in more identified peptides.

Best,

Fengchao
